Description: Force linking to mysqlclient_r to avoid symbol conflicts.
 apr-util's mysql driver is linked against that version of the library
 but due to missing proper symbols versioning we are forced to link to
 the re-entrant library too.
Origin: other, http://bugs.debian.org/469081
Forwarded: not-needed
Last-Update: 2010-01-18

--- a/ext/mysql/config.m4
+++ b/ext/mysql/config.m4
@@ -76,7 +76,7 @@ elif test "$PHP_MYSQL" != "no"; then
 Note that the MySQL client library is not bundled anymore!])
   fi
 
-  if test "$enable_maintainer_zts" = "yes"; then
+  if true || test "$enable_maintainer_zts" = "yes"; then
     MYSQL_LIBNAME=mysqlclient_r
   else
     MYSQL_LIBNAME=mysqlclient
--- a/ext/mysqli/config.m4
+++ b/ext/mysqli/config.m4
@@ -25,7 +25,7 @@ elif test "$PHP_MYSQLI" != "no"; then
     MYSQL_LIB_CFG='--libmysqld-libs'
     dnl mysqlnd doesn't support embedded, so we have to add some extra stuff
     mysqli_extra_sources="mysqli_embedded.c"
-  elif test "$enable_maintainer_zts" = "yes"; then
+  elif true || test "$enable_maintainer_zts" = "yes"; then
     MYSQL_LIB_CFG='--libs_r'
     MYSQL_LIB_NAME='mysqlclient_r'
   else
--- a/ext/pdo_mysql/config.m4
+++ b/ext/pdo_mysql/config.m4
@@ -55,7 +55,7 @@ if test "$PHP_PDO_MYSQL" != "no"; then
       if test "x$SED" = "x"; then
         AC_PATH_PROG(SED, sed)
       fi
-      if test "$enable_maintainer_zts" = "yes"; then
+      if true || test "$enable_maintainer_zts" = "yes"; then
         PDO_MYSQL_LIBNAME=mysqlclient_r
         PDO_MYSQL_LIBS=`$PDO_MYSQL_CONFIG --libs_r | $SED -e "s/'//g"`
       else
